app/src/main/java/org/thoughtcrime/securesms/database/Storage.kt

```diff
@@ -581,7 +581,13 @@ class Storage(context: Context, helper: SQLCipherOpenHelper) : Database(context,
             recipientDatabase.setProfileSharing(recipient, true)
             recipientDatabase.setRegistered(recipient, Recipient.RegisteredState.REGISTERED)
             // create Thread if needed
-            threadDatabase.getOrCreateThreadIdFor(recipient)
+            val threadId = threadDatabase.getOrCreateThreadIdFor(recipient)
```

Comment:
@hjubb this is where we handle config sync fields

Reply:
ahh that makes sense :smile: awesome!